components/button/button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Children } from "react";
+
+interface ButtonProperties {
+  children: React.ReactNode;
+  backgroundURL: string;
+  height: string;
+  width: string;
+  onClick: () => void;
+}
+
+const Button: React.FC<ButtonProperties> = ({ 
+  children, 
+  backgroundURL, 
+  height,
+  width,
+  onClick 
+}) => {
+  return (
+    <div className="container">
```

Comment:
let's rename this to be `custom-button-container` to be more specific because i imagine we'll be having lots on containerized components